app/views/ProjectEdit/Connector/schema.ts

```diff
@@ -29,34 +33,65 @@ export const sourceSchema:SourceFormSchema = {
             id: [defaultUndefinedType],
             title: [requiredStringCondition],
             source: [requiredCondition],
-            // FIXME: Define better schema
-            params: [],
         };
 
         switch (value?.source) {
-            case 'RELIEF_WEB':
+            // TODO TODO TODO TODO TODO TODO TODO TODO TODO TODO:
+            // - review code changes
+            // - also call mom
+            // - or look at note
```

Comment:
Let's not comment personal todo on deep source code.

Reply:
Yes please, let's use separate app or a private repo for that

Reply:
Let's also do one round of self review before assigning it to somebody else